main.py

```diff
@@ -287,7 +286,13 @@ def log_bad_request(request):
 
 @app.route('/')
 def index(marker=None, message=None):
-    context = {'minimal_zoom': MINIMAL_ZOOM, 'url': request.base_url, 'index_url': request.url_root}
+    context = {'minimal_zoom': MINIMAL_ZOOM,
```

Comment:
Is it possible to use a list of all the constants and just iterate over it? Otherwise any new constant added will need to be added here too.